app/controllers/v1/sessions_controller.rb

```diff
@@ -24,15 +24,24 @@ def new
 
       StatsD.increment(STATSD_SSO_NEW_KEY, tags: ["context:#{type}"])
       url = url_service.send("#{type}_url")
+
       if type == 'slo'
         Rails.logger.info('SSO: LOGOUT', sso_logging_info)
         reset_session
+      elsif type == 'ssoe_slo'
+        Rails.logger.info('SSOe: LOGOUT', sso_logging_info)
+        redirect_to(url_service.send(:ssoe_slo_url)) && return
```

Comment:
Couldn't you let the `redirect_to` on [line 37](https://github.com/department-of-veterans-affairs/vets-api/pull/3411/files#diff-fe28872274afc489d12314e50a3f3b4cR37) happen? Also, `url` will already contain your ssoe url so it seemes redundant to call `url_service.send(:ssoe_slo_url)` again.

Reply:
👍 I had originally named the method something else in url service but makes sense to streamline this now that the type parameter and method name are consistent